internal/common/disasters/environmentdefinition.go

```diff
@@ -69,3 +69,21 @@ func (e Environment) DisasterEffects() map[shared.ClientID]float64 {
 	}
 	return out
 }
+
+// DisasterEffects returns the porportional effects of the most recent DisasterReport held in the environment state
```

Comment:
```suggestion
// DisasterEffects returns the proportional effects of the most recent DisasterReport held in the environment state
```
Note spelling of proportional :)